src/client/tile.h

```diff
@@ -154,6 +154,7 @@ video::IImage * Align2Npot2(video::IImage * image, video::IVideoDriver* driver);
 #endif
 
 enum MaterialType{
+	TILE_MATERIAL_OPAQUE,
```

Comment:
Since this is an enum, the added material should be added at the end i think, to not change the enumerations of the others.